tensorboard/webapp/widgets/card_fob/card_fob_test.ts

```diff
@@ -52,13 +54,17 @@ describe('card fob', () => {
 
   function createFobComponent(input: {
     step?: number;
+    allowRemoval?: boolean;
     axisDirection?: AxisDirection;
   }): ComponentFixture<TestableFobComponent> {
     const fixture = TestBed.createComponent(TestableFobComponent);
     fixture.componentInstance.step = input.step ? input.step : 1;
     fixture.componentInstance.axisDirection = input.axisDirection
       ? input.axisDirection
       : AxisDirection.HORIZONTAL;
+    if (input.allowRemoval !== undefined) {
+      fixture.componentInstance.allowRemoval = Boolean(input.allowRemoval);
```

Comment:
why you need to cast to Boolean? Isn't the type specified in param already?

Reply:
That's a good point. Originally I was planning on just passing the input directly through but the default to true made that kinda funky.